addons/compat_ace_fire/config.cpp

```diff
@@ -0,0 +1,30 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"diw_armor_plates_main",QUOTE(REPLACEMOD)};
+        author = "Alien314";
+        authors[] = {"Alien314","ACE-Team"};
+        url = "";
+        VERSION_CONFIG;
+        skipWhenMissingDependencies = 1;
+    };
+};
+
+#include "ACE_Medical_injuries.hpp"
```

Comment:
```suggestion
#include "ACE_Medical_Injuries.hpp"
```

Works on windows, but not on linux

Reply:
Oh, that's why the lint thing was missing that